init.sql

```diff
@@ -215,30 +268,35 @@ COPY public.transcripts (id, content, "originalContent", status, "archivedAt", "
 --
 
 COPY public.users (id, "githubUsername", "authToken", permissions, "archivedAt", "createdAt", "updatedAt") FROM stdin;
```

Comment:
so there will be no user's in the seed? how will we grant admin permissions to anyone from a fresh seed?

Reply:
i have updated this to include users from [here](https://queue.btctranscripts.com/api/users)